core/src/main/scala/kafka/admin/ConsumerGroupCommand.scala

```diff
@@ -55,77 +56,132 @@ object ConsumerGroupCommand {
 
     val consumerGroupService = {
       if (opts.useOldConsumer) {
+        println("Note: This will only show information about consumers that use ZooKeeper (not those using the Java consumer API).\n")
         new ZkConsumerGroupService(opts)
       } else {
+        println("Note: This will only show information about consumers that use the Java consumer API (non-ZooKeeper-based consumers).\n")
         new KafkaConsumerGroupService(opts)
       }
     }
 
     try {
       if (opts.options.has(opts.listOpt))
-        consumerGroupService.list()
-      else if (opts.options.has(opts.describeOpt))
-        consumerGroupService.describe()
+        consumerGroupService.listGroups().foreach { printMessage(_) }
```

Comment:
nitpick: I think parenthesis are a little nicer for simple one-liners like this.